internal/store/agents.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package store
+
+import (
+	"context"
+	"fmt"
+
+	om "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
+	"github.com/mongodb/mongocli/internal/config"
+)
+
+type AgentLister interface {
+	Agents(string, string) (*om.Agents, error)
+}
+
+type AgentStore interface {
```

Comment:
we can do without this interface since is the same as `AgentLister`, don't forget to regenerate the mocks

Reply:
Yes but we may decide to add also the other agent api call in the future 😄

Reply:
okay, we will add the interface when we will add the other call 🙄

Reply:
then that person can add the store in their PR, anyway this is nothing major so I won't block based on this just a goos practice to avoid [Yagni](https://martinfowler.com/bliki/Yagni.html)